nepalingo-web/package.json

```diff
@@ -10,8 +10,12 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@supabase/supabase-js": "^2.44.2",
+    "install": "^0.13.0",
+    "pnpm": "^9.4.0",
```

Comment:
`pnpm` and `install` should not be added here. What is `install` for?

There are two different kinds of dependencies in packages.json

dependencies are code that are needed to run the app

devDependencies are code that are needed to develop the app

in this case pnpm isn't needed to run the code on the client so that could be added as a dev dependency

Reply:
Delete 
``` "install": "^0.13.0",
    "pnpm": "^9.4.0",```

rerun `pnpm install`

Reply:
done